### ISSUE TITLE
[FEATURE] hide the charging bolt when the battery level reaches 100 just like ios 16.

### ISSUE THREAD

**Issue by saeedbalkhair** [MTACS/Ampere#8]
Also, I was wondering whether you can add a toggle to turn (on/off) Percentage always shown inside (Control Center).

**Comment by Dr-Sauce** [comment]
Yes, it would be great if the lighting bolt disappears when the battery is fully charged. (Currently 100 and the lighting blot can’t fit in the battery icon)

And making a toggle to disable the iOS 16 style battery icon in control centre would be great. iOS 16 shows the battery level text next to the battery icon in control centre.

**Comment by MTACS** [comment]
Added the option to hide the bolt at 100% in the next update and will work on the toggle for CC

**Comment by saeedbalkhair** [comment]
> Added the option to hide the bolt at 100% in the next update and will work on the toggle for CC

Thank you and one last thing if it is possible to resize the charging bolt to a bigger size like iOS 16. 🙏🏼

**Comment by Dr-Sauce** [comment]
@MTACS Thank you a lot..! 🙏🙏
